scala/pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.microsoft.spark</groupId>
   <artifactId>spark-clr</artifactId>
-  <version>${spark.version}-SNAPSHOT</version>
+  <version>1.4.1-SNAPSHOT</version>
```

Comment:
What is the warning? If it is not serious we may want to keep the macro as it helps keeping version in one place in the file.